app/src/main/res/drawable/ic_done.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/res/drawable/ic_done.xml#L1"></span>The resource <code>R.drawable.ic_done</code> appears to be unused</td>
    </tr>
  </tbody>
</table>